plenum/client/client.py

```diff
@@ -286,8 +286,7 @@ def submitReqs(self, *reqs: Request) -> List[Request]:
 
         for request in reqs:
             is_read_only = request.txn_type in self._read_only_requests
-            if (self.mode == Mode.discovered and self.hasSufficientConnections) or \
-               (self.hasAnyConnections and (is_read_only or request.isForced())):
+            if self.can_send_request(request):
                 recipients = self._connected_node_names
                 if is_read_only and len(recipients) > 1:
```

Comment:
Can can_send_read_requests be used here?

Reply:
`can_send_request` handles both cases (read and write).